data/persistence/src/commonMain/kotlin/social/androiddev/common/persistence/localstorage/DodoLocalStorage.kt

```diff
@@ -0,0 +1,6 @@
+package social.androiddev.common.persistence.localstorage
+
+class DodoLocalStorage {
```

Comment:
interface for all the things please. This way we can get around even needing the TODO, for now we can have interface DodoLocalStorage

Reply:
Yeah agreed, will be updating in next commits.